sippy-ng/src/component_readiness/ComponentReadiness.js

```diff
@@ -0,0 +1,861 @@
+  }
+  const [excludeNetworksCheckedItems, setExcludeNetworksCheckedItems] =
+    React.useState(initExcludeNetworksList)
+  //console.log('initExcludeNetworksList: ', initExcludeNetworksList)
+  console.log('excludeNetworksCheckedItems: ', excludeNetworksCheckedItems)
+
+  tmp = groupByParameters.get('baseRelease')
+  let initBaseRelease = '4.14'
+  if (tmp != null) {
+    initBaseRelease = tmp
+  }
+  const [baseRelease, setBaseRelease] = React.useState(initBaseRelease)
+
+  tmp = groupByParameters.get('sampleRelease')
+  let initSampleRelease = '4.13'
+  if (tmp != null) {
+    initSampleRelease = tmp
+  }
+  const [sampleRelease, setSampleRelease] = React.useState(initSampleRelease)
+
+  tmp = groupByParameters.get('baseStartTime')
+  let initBaseStartTime = initialStartTime
+  if (tmp != null) {
+    initBaseStartTime = tmp
+  }
+  const [baseStartTime, setBaseStartTime] = React.useState(initBaseStartTime)
+
+  tmp = groupByParameters.get('baseEndTime')
+  let initBaseEndTime = initialEndTime
+  if (tmp != null) {
+    initBaseEndTime = tmp
+  }
+  const [baseEndTime, setBaseEndTime] = React.useState(initBaseEndTime)
+
+  tmp = groupByParameters.get('sampleStartTime')
+  let initSampleStartTime = initialPrevStartTime
+  if (tmp != null) {
+    initSampleStartTime = tmp
+  }
+  const [sampleStartTime, setSampleStartTime] =
+    React.useState(initSampleStartTime)
+
+  tmp = groupByParameters.get('sampleEndTime')
+  let initSampleEndTime = initialPrevEndTime
+  if (tmp != null) {
+    initSampleEndTime = tmp
+  }
+  const [sampleEndTime, setSampleEndTime] = React.useState(initSampleEndTime)
+
+  const excludeUpgradesList = [
+    'no-upgrade',
+    'none',
+    'upgrade-micro',
+    'upgrade-minor',
+  ]
+
+  tmp = groupByParameters.get('exclude_upgrades')
+  let initExcludeUpgradesList = []
+  if (tmp !== null && tmp !== '') {
+    initExcludeUpgradesList = tmp.split(',')
+  }
+  const [excludeUpgradesCheckedItems, setExcludeUpgradesCheckedItems] =
+    React.useState(initExcludeUpgradesList)
+  //console.log('initExcludeUpgradesList: ', initExcludeUpgradesList)
+  console.log('excludeUpgradesCheckedItems: ', excludeUpgradesCheckedItems)
+
+  const excludeVariantsList = [
+    'assisted',
+    'compact',
+    'fips',
+    'hypershift',
+    'microshift',
+    'osd',
+    'proxy',
+    'rt',
+    'serial',
+    'single-node',
+    'standard',
+    'techpreview',
+  ]
+  tmp = groupByParameters.get('exclude_variants')
+  let initExcludeVariantsList = []
+  if (tmp !== null && tmp !== '') {
+    initExcludeVariantsList = tmp.split(',')
+  }
+  const [excludeVariantsCheckedItems, setExcludeVariantsCheckedItems] =
+    React.useState(initExcludeVariantsList)
+  //console.log('initExcludeVariantsList: ', initExcludeVariantsList)
+  console.log('excludeVariantsCheckedItems: ', excludeVariantsCheckedItems)
+
+  const pageTitle = (
+    <Typography variant="h4" style={{ margin: 20, textAlign: 'center' }}>
+      Component Readiness for {baseRelease}
```

Comment:
WDYT about making this `Component Readiness for {baseRelease} vs. {sampleRelease}`?